src/main/kotlin/no/nav/familie/ef/iverksett/arbeidsoppfolging/ArbeidsoppfølgingMapper.kt

```diff
@@ -0,0 +1,38 @@
+package no.nav.familie.ef.iverksett.arbeidsoppfolging
+
+import no.nav.familie.ef.iverksett.iverksetting.domene.IverksettOvergangsstønad
+import no.nav.familie.ef.iverksett.iverksetting.domene.VedtaksdetaljerOvergangsstønad
+import no.nav.familie.eksterne.kontrakter.arbeidsoppfolging.Aktivitetstype
+import no.nav.familie.eksterne.kontrakter.arbeidsoppfolging.Barn
+import no.nav.familie.eksterne.kontrakter.arbeidsoppfolging.Periode
+import no.nav.familie.eksterne.kontrakter.arbeidsoppfolging.Periodetype
+import no.nav.familie.eksterne.kontrakter.arbeidsoppfolging.Stønadstype
+import no.nav.familie.eksterne.kontrakter.arbeidsoppfolging.VedtakOvergangsstønadArbeidsoppfølging
+import no.nav.familie.eksterne.kontrakter.arbeidsoppfolging.Vedtaksresultat
+
+object ArbeidsoppfølgingMapper {
+
+    fun mapTilVedtakOvergangsstønadTilArbeidsoppfølging(
+        iverksett: IverksettOvergangsstønad,
+    ): VedtakOvergangsstønadArbeidsoppfølging {
+        return VedtakOvergangsstønadArbeidsoppfølging(
+            vedtakId = iverksett.behandling.eksternId,
+            personIdent = iverksett.søker.personIdent,
+            barn = iverksett.søker.barn.map { Barn(it.personIdent, it.termindato) },
+            stønadstype = Stønadstype.valueOf(iverksett.fagsak.stønadstype.name),
+            vedtaksresultat = Vedtaksresultat.valueOf(iverksett.vedtak.vedtaksresultat.name),
```

Comment:
Burde vi legge til en test for mapping av enums? Se `FellesTilEksterneKontrakterEnumTest`

Reply:
Ja, den er kjekk! La til som egen test for å skille på det som gjaldt arbeidsoppfølging.